src/binding/python/RecordComponent.cpp

```diff
@@ -279,44 +280,55 @@ store_chunk(RecordComponent & r, py::array & a, Offset const & offset, Extent co
         throw py::index_error(
             "strides in chunk are inefficient, not implemented!");
     // @todo in order to implement stride handling, one needs to
-    //       loop over the input data strides during write below,
-    //       also data might not be owned!
+    //       loop over the input data strides during write below
+
+    // check: did the user pass a temporary buffer? In that case, we copy as we
+    //        do in the shared_ptr API for C++
+    // note: there is a corner-case if the user passes a temporary via additional
```

Comment:
is->might be ... not sure if that would not decrease the refcount again once passed as an arg